tests/functional/test_all.py

```diff
@@ -252,7 +253,15 @@ def test_execution_plan(self):
 
         result = redis_graph.execution_plan("MATCH (r:Rider)-[:rides]->(t:Team) WHERE t.name = $name RETURN r.name, t.name, $params", {'name': 'Yehuda'})
         expected = "Results\n    Project\n        Conditional Traverse | (t:Team)->(r:Rider)\n            Filter\n                Node By Label Scan | (t:Team)"
-        self.assertEqual(result, expected)
+        self.assertEqual(str(result), expected)
```

Comment:
Please add additional tests for execution-plan construction some a bit more complex.
e.g. multiple operations within the same level

Reply:
done